eppoclient/configurationstore.go

```diff
@@ -1,15 +1,12 @@
 package eppoclient
 
 import (
-	"encoding/json"
 	"errors"
-	"log"
-
-	lru "github.com/hashicorp/golang-lru"
+	lru "github.com/hashicorp/golang-lru/v2"
 )
 
 type configurationStore struct {
-	cache *lru.Cache
+	cache *lru.Cache[string, experimentConfiguration]
```

Comment:
The new `lru.Cache` has support for generics so we instantiate it with the exact types we expect.